portfolio/src/main/webapp/script.js

```diff
@@ -48,25 +48,24 @@ function bubbleFall() {
     }
 }
 
-// Get messages from the data servlet.
-function getMessages() {
-  fetch('/data').then(response => response.json()).then((messages) => {
-    console.log(messages);
+// Get comments from the data servlet.
+function getComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
```

Comment:
I'm assuming that comments is an `Array<string>` but I'm not completely sure because I know that json can take many different forms like a map. You might want to add parameter and return types here (but it's not mandatory).

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments